docs/user/build_docker_with_ssh.md

```diff
@@ -0,0 +1,27 @@
+# Build Docker Images with SSH
```

Comment:
can you merge this with /job_docker_env.md?

Reply:
I added a section in job_docker_env.md. Please check it [here](https://github.com/microsoft/pai/blob/b5f17fb7516b064a36d7db99bb07de415833bd08/docs/job_docker_env.md#enable-ssh-for-your-image).

Reply:
I added a section in job_docker_env.md. Please check it [here](https://github.com/microsoft/pai/blob/b5f17fb7516b064a36d7db99bb07de415833bd08/docs/job_docker_env.md#enable-ssh-for-your-image).